gum/gum.c

```diff
@@ -506,6 +509,18 @@ gum_on_log_message (const gchar * log_domain,
 
     return;
   }
+#else
```

Comment:
Why sometimes the `#if defined` and sometimes the `# ifdef`?
And why sometimes the `#elif` and sometimes the `#else # ifdef`?